src/Identity/UI/src/build/Microsoft.AspNetCore.Identity.UI.props

```diff
@@ -0,0 +1,31 @@
+<Project>
+  <PropertyGroup>
+    <IdentityUIFrameworkVersion Condition="'$(IdentityUIFrameworkVersion)' == ''">Bootstrap4</IdentityUIFrameworkVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AssemblyAttribute Include="Microsoft.AspNetCore.Identity.UI.UIFrameworkAttribute">
+      <_Parameter1>$(IdentityUIFrameworkVersion)</_Parameter1>
+    </AssemblyAttribute>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IdentityUIFrameworkVersion)' == 'Bootstrap3'">
+    <StaticWebAsset Include="$(MSBuildThisFileDirectory)..\staticwebassets\V3\**">
+      <SourceType>Package</SourceType>
+      <SourceId>Microsoft.AspNetCore.Identity.UI</SourceId>
+      <ContentRoot>$([MSBuild]::NormalizePath('$(MSBuildThisFileDirectory)..\staticwebassets\V3'))</ContentRoot>
+      <BasePath>/Identity</BasePath>
+      <RelativePath>%(RecursiveDir)%(FileName)%(Extension)</RelativePath>
+    </StaticWebAsset>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IdentityUIFrameworkVersion)' == 'Bootstrap4'">
+    <StaticWebAsset Include="$(MSBuildThisFileDirectory)..\staticwebassets\V4\**">
+      <SourceType>Package</SourceType>
+      <SourceId>Microsoft.AspNetCore.Identity.UI</SourceId>
+      <ContentRoot>$([MSBuild]::NormalizePath('$(MSBuildThisFileDirectory)..\staticwebassets\V4'))</ContentRoot>
+      <BasePath>/Identity</BasePath>
+      <RelativePath>%(RecursiveDir)%(FileName)%(Extension)</RelativePath>
+    </StaticWebAsset>
+  </ItemGroup>
+</Project>
```

Comment:
I plan to individually enumerate the files here so that we can have a some extra options.
* Do not publish assets if identity is configured to use the app layout. (We can have a pretty good guess with `Exists(
Areas\Identity\Pages\_ViewStart.cshtml')`
* Only publish minified assets by default.
* Disable publishing any static asset